py-scoring/src/ai/h2o/sparkling/Initializer.py

```diff
@@ -173,10 +173,11 @@ def __add_url_to_classloader(gateway, url):
         logger = Initializer.__get_logger(jvm)
         while loader:
             try:
+                urlClassLoaderClass = jvm.py4j.reflection.ReflectionUtil.classForName("java.net.URLClassLoader")
                 classClass = gateway.jvm.Class
                 classArray = gateway.new_array(classClass, 1)
                 classArray[0] = url.getClass()
-                method = loader.getClass().getDeclaredMethod("addURL", classArray)
+                method = urlClassLoaderClass.getDeclaredMethod("addURL", classArray)
                 method.setAccessible(True)
```

Comment:
if addURL was public we could just use `loader.getMethod`, correct? is this why we need to look up the method on URLClassLoader directly?